### PR TITLE
[ENG-793] feat: Add Copper connector

### PR DESCRIPTION
**Pull request by Listener430**
## Catalog variables
No required catalog variables

### GET 
URL: <localhost:4444/account> 
Copper proxy get ok

### POST
URL: <localhost:4444/people>
Copper proxy post ok

### PUT
URL: <localhost:4444/people/160213588>
Cooper proxy put ok

### DELETE
URL: <localhost:4444/people/160213590> 
Copper proxy del ok

## Pagination
Copper_pagination_example

### CODE REVIEW

**Comment by Listener430** [comment]
@RajatPawar, hi, as agreed, sending Copper Connector code separately. Please kindly take a look.

**Comment by Listener430** [comment]
@RajatPawar please kindly take a look when you have a second.

File: providers/catalog.go [hunk 1]
```diff
@@ -156,6 +157,25 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 		},
 	},
 
+	// Copper configuration
+	Copper: {
+		AuthType: Oauth2,
+		BaseURL:  "https://api.copper.com/developer_api/v1",
```
Comment (RajatPawar) [inline providers/catalog.go:163]: Can you please not add the version in the Base URL? If they release a new version, we'll have to update it.

**Review by RajatPawar — CHANGES_REQUESTED** [review]
Can you please not add the version in the Base URL? If they release a new version, we'll have to update it.

File: providers/catalog.go (hunk 1 above)
Comment (RajatPawar) [inline providers/catalog.go:163]: @Listener430 can you please remove /v1 from the base URL? We don't want to have to update this if they release a v2.

**Review by RajatPawar — CHANGES_REQUESTED** [review]
[For https://github.com/amp-labs/connectors/pull/109#pullrequestreview-1948601319](https://github.com/amp-labs/connectors/pull/109#pullrequestreview-1948601319)

**Review by RajatPawar — APPROVED** [review]

**Pull request merged**